axlearn/experiments/text/gpt/fuji.py

```diff
@@ -403,4 +401,24 @@ def make_grad_accum_config(
                 make_grad_accum_config, make_single_host_config_func, 4
             )
 
+        if model_size == "70B":
+
+            def make_config_with_act_offload(base_config_name: str) -> SpmdTrainer.Config:
+                """Make configs for the v5e/v6e tpu with low HBM."""
+                # pytype: disable=annotation-type-mismatch
+                cfg: SpmdTrainer.Config = config_map[base_config_name]().clone()
+                # pytype: enable=annotation-type-mismatch
+                update_model_remat_config(
```

Comment:
Would it call update_model_remat_config twice this way?

Reply:
unfortunately yes, but those are light weight config changes.

Reply:
hmm as long as it doesn't cause issues, it may be fine?